docs/tutorial/part-two/index.md

```diff
@@ -486,16 +297,33 @@ export default () => (
 )
 ```
 
+> Tip: Generally, if you use a component in multiple places on a site, it should be in its own module file in the `components` directory. But, if it's used only in one file, create it inline.
```

Comment:
I'm not sure we need this tip. Might spark some bikeshedding, and could also add to overwhelm ("I have to think about _that_, too?!").

Reply:
I relocated it, it was already there. Personally I think it's useful -- if someone is like "wait why is it in the same file? i haven't seen that in other places!"

what we could do is remove the "suggestion" element, and just say you can do it multiple ways.